dist/grafana/grafana-dashboard.configmap.yaml.tpl

```diff
@@ -0,0 +1,13 @@
+# WARNING: This is generated from dist/grafana/
+# please modify there and run `just dashboards`
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: GRAFANA_METADATA
+  labels:
+    grafana_dashboard: "true"
+  annotations:
+    grafana-folder: /grafana-dashboard-definitions/Cincinnati
+data:
+  cincinnati.json: |-
+GRAFANA_MANIFEST
```

Comment:
nit: can we indent `GRAFANA_MANIFEST` here and then adjust the `sed` to preserve that indentation when injecting the JSON?  That seems like it would be slightly less magical.